sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/AssetConversionOptions.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.MixedReality.ObjectAnchors
+{
+    using Azure.Core;
+    using System;
+
+    /// <summary>
+    /// Represents an Object Anchors asset conversion job request's options.
+    /// </summary>
+    public class AssetConversionOptions
+    {
+        /// <summary>
+        /// Gets the asset configuration values.
+        /// </summary>
+        internal AssetConversionConfiguration ConversionConfiguration { get; }
+
+        /// <summary>
+        /// The file type of the asset.
+        /// </summary>
+        public AssetFileType InputAssetFileType { get; }
+
+        /// <summary>
+        /// The path to the Asset to be ingested by the Object Anchors service.
+        /// </summary>
+        public Uri InputAssetUri { get; }
+
+        /// <summary>
+        /// Identifier for the Object Anchors Asset Conversion job.
+        /// </summary>
+        public Guid JobId { get; set; } = Guid.NewGuid();
+
+        /// <summary> Gravity vector with respect to object's nominal position. </summary>
```

Comment:
You keep mixing different styles of the `<summary>` tag. Let's choose one and stick with it. I'm a bigger fan of having the tab on separate lines myself.

```suggestion
        /// <summary>
        /// Gravity vector with respect to object's nominal position.
        /// </summary>
```

Reply:
Change to multi line summaries pending

Reply:
Multi-line summaries added